hawc/apps/epiv2/views.py

```diff
@@ -11,22 +17,226 @@ class DesignCreate(BaseCreate):
     model = models.Design
     form_class = forms.DesignForm
 
+    def get_success_url(self):
+        return reverse("epiv2:design_update", args=[self.object.id])
+
 
 class DesignUpdate(BaseUpdate):
     success_message = "Study-population updated."
     parent_model = Study
     parent_template_name = "study"
     model = models.Design
     form_class = forms.DesignForm
+    template_name = "epiv2/design_update.html"
+
+    def get_queryset(self):
```

Comment:
can this be moved to a manager or queryset method so we don't have it in two different views? 
https://ccbv.co.uk/projects/Django/4.0/django.views.generic.edit/UpdateView/